Tasks/b0_linear_search.py

```diff
@@ -10,16 +10,10 @@ def min_search(arr) -> int:
 	:param arr: Array containing numbers
 	:return: index of first occurrence of minimal element in array
 	"""
-
-	return -1
-
-
-def min_weight_search(Graph) -> tuple:
-	"""
-	Function that find edge in graph with minimal weight of all
-
-	:param Graph: NetworkX Graph (or digraph) with weighted edges
-	:return: tuple of nodes (node, node) the weight of edge between which is minimal (any occurrence)
-	"""
-
-	return None, None
\ No newline at end of file
+	ind = 0
+	minim = arr[0]
+	for i in range(len(arr)):
```

Comment:
```suggestion
	for i in range(1, len(arr)):
```
Не нужно проверять первый элемент :)